frontend/apps/marketing/src/components/carousels/actionBlockCarousel/ActionBlockCarousel.tsx

```diff
@@ -0,0 +1,133 @@
+'use client';
+
+import {BaseEntry, EntryFields} from 'contentful';
+import '@code-dot-org/component-library/carousel/index.css';
+import React, {ReactNode, useMemo} from 'react';
+
+import ActionBlock, {
+  ActionBlockProps,
+} from '@code-dot-org/component-library/actionBlock';
+import DSCOCarousel from '@code-dot-org/component-library/carousel';
+
+export type ActionBlockCarouselProps = ActionBlockProps & {
```

Comment:
I think we can remove `ActionBlockProps` here as from what I see we only use slides and background prop which both are defined here in code

Reply:
When I remove it I need to define the background type in the component prop (see below) to fix the type error. Seems easier to keep it how it is, but is there a reason not to do it that way?
```
background={
  background === 'primary' || background === 'secondary'
  ? background
    : undefined
}
```

Reply:
It's better to update background types specifically manually, in contentful - make a radio button/dropdown select with options that we accept and then update the type to match what's in contentful